backend/migrations/cmd/migrate-group-embeddings/main.go

```diff
@@ -11,6 +11,8 @@ import (
 	"github.com/highlight-run/highlight/backend/model"
 )
 
+var step = 100000
```

Comment:
Nit: not sure if go has the same convention, but like keeping constants all capitalized to help with readability. If this is in fact a constant.

Reply:
I should change var -> const, but in go unexported constants are typically lowercase / camel case to avoid being exported